interfaces/Portalicious/src/index.html

```diff
@@ -13,19 +13,6 @@
       type="image/x-icon"
       href="assets/favicon.ico"
     />
-    <link
-      rel="preconnect"
-      href="https://fonts.googleapis.com"
-    />
-    <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-      crossorigin
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
-      rel="stylesheet"
-    />
```

Comment:
Can we not break the fonts? I understand the reasoning in the task you've linked, but it will be incredibly annoying to work on a broken website in the meantime.

If introducing other changes relies on the changes in this file, then I would recommend doing them in conjunction with fixing the font loading, rather than breaking the fonts.